.unreleased/LLT-5291

```diff
@@ -0,0 +1 @@
+Make the body of events non-optional
```

Comment:
idk if this affects library users, so not sure if this is actually needed :shrug:

Reply:
It dies affect, we add do it to also improve udl